config/flags_service_start.go

```diff
@@ -69,6 +69,13 @@ var (
 		Usage: "Comma separated list of active services.",
 		Value: strings.Join([]string{"wireguard", "scraping", "data_transfer"}, ","),
 	}
+
+	// FlagStoppedServices a comma-separated list of stopped services.
+	FlagStoppedServices = cli.StringFlag{
+		Name:  "stopped-services",
+		Usage: "Comma separated list of stopped services.",
```

Comment:
Why do we need an extra flag for stopped services? All services without active services already list stopped services, right?